recipes-core/images/openwrt-image-base.bb

```diff
@@ -0,0 +1,48 @@
+SUMMARY = "OpenWrt Image Minimal"
```

Comment:
Will rename the image recipe. Should the LICENSE be changed from "CLOSED" to "MIT"? Didn't change that since the commit came from OSSystems fork.